libp2p/crypto/ecc.py

```diff
@@ -32,7 +32,7 @@ def get_type(self) -> KeyType:
         return KeyType.ECC_P256
 
     def verify(self, data: bytes, signature: bytes) -> bool:
-        raise NotImplementedError
+        raise NotImplementedError()
```

Comment:
Actually `raise NotImplementedError` works: https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement

Reply:
Huh, thanks for telling. I personally like putting braces after them (to be more explicit), but I'll reverse these changes if that's requested.